web/src/ApiButton.tsx

```diff
@@ -410,19 +411,40 @@ export function ApiButton(props: React.PropsWithChildren<ApiButtonProps>) {
   }
 }
 
+// UIButtons for a location, sorted into types
+export type ButtonSet = {
+  normal: UIButton[]
+  toggleDisable?: UIButton
+}
```

Comment:
i find `normal` a little odd/confusing. what do you think about `default` or `userDefined`?